tests/regression/nextcloud-rule-exclusions-plugin/9508701.yaml

```diff
@@ -0,0 +1,23 @@
+---
+meta:
+  author: "Esad Cetiner"
+  description: "Nextcloud Rule Exclusions Plugin"
+  enabled: true
+  name: 9508701.yaml
+tests:
+  - test_title: 9508701-1
+    desc: Dismissing notifications on iOS
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: OWASP CRS test agent
+              Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
+            port: 80
+            method: DELETE
+            uri: /ocs/v2.php/apps/notifications/api/v2/push
+            version: HTTP/1.1
+          output:
+            no_log_contains: id "911100"
```

Comment:
950100?

Reply:
Oops. I had scrolled too far down. I thought you were disabling rule 950100, but that's already the next rule.